### ISSUE TITLE
Search Input Box Outline A11y

### ISSUE THREAD

**Issue by hwkr** [scalar/scalar#5005]
### What happens?

Reported:

> I noticed in this release there is a border shadow around the sidebar-search input field that shows up differently in different browsers. The screenshot is on our site, but i see the same behavior on sandbox.scalar.com. This line is controlled by the css attribute box-shadow.
>  
> This shows safari, chrome, and firefox from left to right.
> ![Image](https://github.com/user-attachments/assets/232a3e2f-a91d-413b-a5a8-4f8bb482b69b)

### What did you expect to happen?

Should be consistent across browsers.
